docs/data/material/getting-started/templates/dashboard/components/StatCard.tsx

```diff
@@ -36,49 +34,64 @@ export default function StatCard({
   trend,
   data,
 }: StatCardProps) {
-  const color = trendColors[trend];
+  const theme = useTheme();
+
+  const trendColors = {
```

Comment:
I made some adjustments to the `StatCard` to fix text contrast and to make the chart colors a bit more balanced on light/dark